src/txn/txn.c

```diff
@@ -714,8 +856,14 @@ __txn_resolve_prepared_op(WT_SESSION_IMPL *session, WT_TXN_OP *op, bool commit,
          * Resolve the prepared update to be committed update.
          */
         __txn_resolve_prepared_update(session, upd);
+        resolved = true;
     }
 
+    /* Fix the history store contents if exists. */
```

Comment:
Should be `Fix the history store contents if they exist`

Reply:
Fixed.